pandas/tests/indexes/datetimes/test_arithmetic.py

```diff
@@ -14,70 +14,109 @@
                     date_range)
 
 
+
 class TestDatetimeIndexArithmetic(object):
     tz = [None, 'UTC', 'Asia/Tokyo', 'US/Eastern', 'dateutil/Asia/Singapore',
           'dateutil/US/Pacific']
 
-    def test_add_iadd(self):
-        for tz in self.tz:
-
-            # offset
-            offsets = [pd.offsets.Hour(2), timedelta(hours=2),
-                       np.timedelta64(2, 'h'), Timedelta(hours=2)]
-
-            for delta in offsets:
-                rng = pd.date_range('2000-01-01', '2000-02-01', tz=tz)
-                result = rng + delta
-                expected = pd.date_range('2000-01-01 02:00',
-                                         '2000-02-01 02:00', tz=tz)
-                tm.assert_index_equal(result, expected)
-                rng += delta
-                tm.assert_index_equal(rng, expected)
-
-            # int
-            rng = pd.date_range('2000-01-01 09:00', freq='H', periods=10,
-                                tz=tz)
-            result = rng + 1
-            expected = pd.date_range('2000-01-01 10:00', freq='H', periods=10,
-                                     tz=tz)
-            tm.assert_index_equal(result, expected)
-            rng += 1
-            tm.assert_index_equal(rng, expected)
+    # Several ways of representing two hours
```

Comment:
you should make this a fixture

Reply:
Just getting comfortable with pytest.parametrize, will look into fixtures.  Mind saving for a follow-up?  I want to avoid making the mistake of having multiple active PRs touching the same modules lest rebase-hell ensue.  Things Learned The Hard Way, Volume 8.

Reply:
well this ends up add and then removing lots of code
would just like to see it down once

Reply:
Fair enough.